js/all-events-template.js

```diff
@@ -54,6 +59,8 @@ function handleData(events) {
       twitterHandle += 'london';
     } else if (document.URL.indexOf('/madrid') !== -1) {
       twitterHandle += 'madrid';
+    } else if (document.URL.indexOf('/tel-aviv') !== -1) {
+      twitterHandle += 'tel-aviv';
```

Comment:
```suggestion
      twitterHandle += 'tlv';
```
Their handle is nodegirlstlv